src/lib/theme/colors.ts

```diff
@@ -112,18 +112,18 @@ const palette: Readonly<Record<string, Record<Shade, HexColor>>> = {
     DEFAULT: '#FF4518',
   },
   slate: {
-    50: '#F5F7F9',
-    100: '#E7EDF2',
-    200: '#D5DEE8',
-    300: '#B9C9D7',
-    400: '#97AEC3',
-    500: '#7E95B3',
-    600: '#667CA1',
-    700: '#607195',
-    800: '#525D7B',
-    900: '#444F64',
-    950: '#2D323E',
-    DEFAULT: '#7E95B3',
+    50: '#E8EFFF',
+    100: '#C9D9F0',
+    200: '#AEBED9',
+    300: '#92A4C3',
+    400: '#7C8FB1',
+    500: '#667CA1',
+    600: '#576E8F',
+    700: '#465A78',
+    800: '#273860',
+    900: '#1E293B',
+    950: '#0F172A',
+    DEFAULT: '#667CA1',
```

Comment:
Colors taken from here: https://www.figma.com/file/OZBSltgZ91xMkooeNWK35L/THE-OOZE-(config-file)?type=design&node-id=1323-606&mode=dev